experimental/dds/tree2/src/feature-libraries/editable-tree-2/proxies/proxies.ts

```diff
@@ -164,11 +167,14 @@ export function createObjectProxy<TSchema extends StructSchema, TTypes extends A
 				if (field !== undefined) {
 					return getProxyForField(field);
 				}
-				// TODO: Do this a better way.
-				if (key === treeNodeSym) {
-					return { schema };
+				switch (key) {
+					case treeNodeSym:
+						return content;
+					case nodeSym:
+						return nodeApi;
+					default:
+						return Reflect.get(target, key);
 				}
-				return undefined;
```

Comment:
Changing 'undefined' to 'Reflect.get(..)' fixes the missing inheritance from 'Object.prototype'...  which is test in another pending PR.